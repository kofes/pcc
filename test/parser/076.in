function foobar(var arr: array of char): boolean;
begin
end;

begin
end
