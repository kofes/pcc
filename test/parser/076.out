<func table>:
<function>:FOOBAR
   <return's type>:
      BOOLEAN
   <params>:
      ARR:v:ARRAY OF CHAR:
   <body>:
   <end>

<type table>:
BOOLEAN
CHAR
INTEGER
REAL
<var table>:
PI:C:REAL:3.1415926535897932385
<block statement>:
------------------
